logspout/Dockerfile

```diff
@@ -1,3 +1,4 @@
+ARG TAG=v3.2.14-disable-logging
```

Comment:
added this so we can use the CI docker build infrastructure

Reply:
also will need to build this in CI before https://github.com/AudiusProject/audius-docker-compose/pull/50 gets merged